README.md

```diff
@@ -1,3 +1,6 @@
+
+# pandas-mentoring
+
```

Comment:
we don't want this, it was in a previous review

Reply:
@datapythonista  I understand and I apologize for any inconvenience this may have caused.